### PR TITLE
Update direct-routing-country-codes.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6823
Amended note as per @dariomws recommendation.

### CODE REVIEW

File: Teams/direct-routing-country-codes.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Set-CSOnlinePSTNGateway -Identity sbc1.contoso.com –MediaRelayRoutingLocationO
 Note that Microsoft only recommends setting this parameter if the call logs clearly indicate that the default assignment of the datacenter for the media path does not use the path closest to the SBC datacenter. 
 
 > [!NOTE]
-> -MediaRelayRoutingLocationOverride - this command is not available for use in direct routing scenarios.
+> -MediaRelayRoutingLocationOverride parameter is reserved for use with managed carriers.
```
Comment (dariomws) [inline Teams/direct-routing-country-codes.md:35]:
```suggestion
> MediaRelayRoutingLocationOverride parameter is reserved for use with managed carriers.
```

**Review by dariomws — COMMENTED** [review]
Let's wait for confirmation about this note before it's merged.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by lindspea** [comment]
> Let's wait for confirmation about this note before it's merged.



> Let's wait for confirmation about this note before it's merged.

Thanks @dariomws who can confirm this?  @scanum please see Dario's note.

**Comment by dariomws** [comment]
@CarolynRowe could you kindly help us to confirm this information? Thanks!

**Comment by dariomws** [comment]
@CarolynRowe could you kindly help us with this PR? Thanks!

**Pull request merged**